maat-court-data-api/src/main/resources/db/changelog/changes/increase_column_various_mla_tables.sql

```diff
@@ -0,0 +1,14 @@
+ALTER TABLE "MLA"."XXMLA_WQ_DEFENDANT" modify FORENAME varchar2(50);
```

Comment:
As per the Jira https://dsdmoj.atlassian.net/browse/LASB-1922 I think the new max length of the fields (FORENAME etc.) should be 100 instead of 50.

> Increase the VARCHAR size to 100 for the XXMLA_WQ_DEFENDANT columns causing errors.

Reply:
I was probably being overly cautious when I put a char limit of 100 in the ticket, there are lots of examples of it being 50 chars in the logs so I would like a bit of wiggle room in the length so maybe 60 would do?

Reply:
@stone-dj please review this story with a few data comments to support this change. https://dsdmoj.atlassian.net/browse/LASB-1922

Reply:
ALTER TABLE "MLA"."XXMLA_WQ_DEFENDANT" modify ADDRESS_LINE1 varchar2(100);
ALTER TABLE "MLA"."XXMLA_WQ_DEFENDANT" modify ADDRESS_LINE2 varchar2(100);
ALTER TABLE "MLA"."XXMLA_OFFENCE" modify OFFENCE_WORDING varchar2(4000);